Formula/alot.rb

```diff
@@ -0,0 +1,101 @@
+class Alot < Formula
+  desc "Terminal-based mail user agent for the notmuch mail system"
+  homepage "https://github.com/pazz/alot"
+  url "https://github.com/pazz/alot/archive/0.3.7.tar.gz"
+  sha256 "fd2d4121ba109224560919e84e320b09526891fbc0038afcea26a3f1284dad26"
+  head "https://github.com/pazz/alot.git", :branch => "testing"
+
+  option "without-docs", "Don't build documentation"
+
+  depends_on "gpgme"
+  depends_on "libmagic"
+  depends_on "notmuch" => "with-python"
```

Comment:
There's two options:

1. Enable installation of the python bindings by default on the `notmuch` formula.
2. The python bindings could be `vendor`'ed in this formula.

What's your preference?